src/commands/generateRunbook.ts

```diff
@@ -17,8 +18,16 @@ const DEFAULT_USER = "local-user";
 
 const prepareRunbookFile = async (workspaceFolder: vscode.WorkspaceFolder, promptType: string) => {
 
+    let friendlyPromptName = promptType;
```

Comment:
Can we have a friendlyPromptName function that we can test and document - it'll be easier for us to describe what other IDE plugins have to do create consistent behavior across different editors.